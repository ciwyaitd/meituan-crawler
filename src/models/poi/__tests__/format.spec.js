'use strict'

describe('models.poi.format', () => {
  it('should match the snap shot when work as expected', () => {
    const poiPayload = require('../../../test/mock_data/poi_payload.json')
    const format = require('../format')
    const result = format(poiPayload)
    expect(result).toMatchSnapshot()
  })
})
