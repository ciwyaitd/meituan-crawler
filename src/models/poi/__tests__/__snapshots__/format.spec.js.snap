// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`models.poi.format should match the snap shot when work as expected 1`] = `
Object {
  "addr": "罗湖区金稻田路边防布心住宅区A座3号4号商铺（近金稻田路公交站）",
  "allow_refund": false,
  "area_id": 29,
  "area_name": "布心/太白路",
  "avg_price": 45,
  "avg_score": 4.2,
  "brand_id": 0,
  "brand_name": "",
  "cate_name": "蛋糕",
  "cates": "1,20557,20426,20097,21009,20639,20428,11,20625,20589",
  "city_id": 30,
  "discount": "3.5折起",
  "feature_menus": "榴莲,芒果,榴莲蛋糕,幸福西饼,吃,榴芒,好,提拉米苏,榴茫双拼,蛋糕四重奏,榴芒蛋糕,榴芒雙拼,芒果蛋糕,一个,芒果慕斯,很好吃,榴莲双拼,芒果茫茫,刀叉,榴芒双拼,新鲜,好好,黑森林蛋糕,质量,白巧克力,四重奏蛋糕,榴莲飘香,很新鲜,双拼蛋糕,心花怒放蛋糕,味道很好,水果蛋糕,奶油蛋糕,芒果拿破仑,磅蛋糕,慕斯,挺好,蛋糕胚,茫茫,蛋糕很漂亮,外包装,榴蓮,工具,4,差,挺好的,芒果千层心蛋糕,芝士蛋糕,芒榴双拼,心花怒放,轻芝士蛋糕,巧克力,蛋糕榴莲,榴莲芒果味,榴莲口味蛋糕,幸福蛋糕,芒果草莓蛋糕,超值,苏格兰榛子,提拉米苏蛋糕,太棒了,芒果千层糕,鲜果嘉年华,千层榴莲饼,蓝莓蛋糕,芒果芒芒慕斯,小蛋糕,好看好吃,老婆饼,巧克力的诱惑,榴莲慕斯蛋糕,蛋挞,good,拿破仑蛋糕,好好的,巧克力牌,猫山王,千层榴莲,蓝莓芝士蛋糕,芒果慕丝,栗子慕斯,草莓慕斯,黑白之恋,芒果栗子蓉,3榴芒双拼,美丽,蛋糕不错,蛋糕可以,举个粟子,可可粉,芒果多,招牌榴莲,榴莲香雪,包装不错,造型好看,新鲜美味,白天使蛋糕,雪顶榴心蛋糕,芒果拿破仑蛋糕,慕斯芒果,草莓多多,四味,巧克力拿破仑,特色,豆浆,粟子蛋糕,小乳酪球,草莓榴莲,榴莲千层酥,榴芒蛋糕慕斯蛋糕,刀叉蜡烛,冰激凌,榴莲雪糕,草莓黑森林蛋糕,草莓夹心,小块蛋糕,水果生日蛋糕,冰激凌栗子蓉,创意蛋糕,榴莲酥,榴莲千层饼,草莓白天使,芒果栗心,轻芝士,蓝莓慕斯,榴莲双芒,芒果芒,拿破仑,提拉米苏黑森林,芒果口味蛋糕,榛子蛋糕,私房蛋糕,黑森林慕斯,果肉蛋糕,水果双拼,饼干,鲜果嘉年华蛋糕,饼皮,榴莲千层榴莲,榴莲芝士蛋糕,8寸蛋糕,纽约芝士,坚果蛋糕,榴恋蛋糕,英式下午茶B款,板栗蛋糕,榴莲马卡龙蛋糕,芒果多多,咖喱牛腩饭,绿豆饼,奶油生日蛋糕,两情相悦,黑白恋曲,馏芒蛋糕,榴莲超级好吃。,实品,G00d,日式芝士蛋糕,蛋糕外包装,榴莲味的比芒果味的做的正,幸福糖,榛子薄脆,榴莲完胜芒果,开箱就败,量多,朱古力卷,草莓黑天使,芒果布丁,芒果奥利奥,很好吃的哦,金牌水果,芒果拼榴莲,蛋糕水果,火龙果,儿童蛋糕,招牌蛋糕,毛毛虫面包,蓝莓夹心,芒果干层,魔方蛋糕,提拉米苏咖啡,苏格兰榛子蛋糕,水果茶,坚果巧克力,雪顶蛋糕,榴莲糕,桃李满天下,榴莲拼提拉米苏,原味蛋糕,提拉米苏慕斯,鲜芒果蛋糕,冰淇淋蛋糕,芒果卷寿司,乳鸽,花之恋寿司,烧鹅,蟹仔寿司,森林果王,长寿面,下午茶拼盘,芒果班戟,芒果慕斯提拉米苏,红豆沙,草莓双拼,香醇咖啡,红烧肉,榴莲饼,布丁冰淇淋,鱿鱼须,鸡公煲,糖油粑粑,蒸排骨,榛子巧克力蛋糕,草莓酸奶,炒米粉,黑森林提拉米苏,千层芒果蛋糕,榴莲干层,芒果草莓双拼,猪肚鸡,配酒,三文鱼刺身,芒果芒果,草莓巧克力,棒棒糖,榴茫双拼榴莲双拼,普通面包,枣泥糕,臻子蛋糕,雪顶榴莲,龙珠双盒,多层蛋糕,水果慕斯蛋,千层蛋糕,牛轧糖,重芝士,葱油鸡,榴茫,水果奶油蛋糕,巧克力威化,缪氏红焖鱼,车厘子,榴莲巧克力蛋糕,提拉米苏榴芒蛋糕,蓝莓巧克力,生日面,四重奏慕斯蛋糕,蛋,四种混合,芒果沙滩,腰果曲奇,夏威夷果曲奇,许愿,咖啡慕斯,幸福冰芝士,手撕微波包,北海道吐司,幸福礼幸福蛋黄白莲蓉,榴芒雙拼h,招牌轻芝士,火山黑森林,味道很好，准时送过来，不错。,芒果雪沙,浓味芝士蛋糕,四种口味的，都挺好吃的。,好香的芒果,迷彩冰芝月饼,金罐曲奇,凤梨酥,黄色糕体不知道什么做的,巧克力牌融了都,帕玛森芝士,颜色好漂亮,蛋糕好吃   大家都喜欢,蛋糕图片,特别的美味！,！,美观好吃,日式芝士,樱桃芝士慕斯,蛋糕巧克力加马卡龙,好吃不是第一次了！,香雪茫茫,好漂酿的打火机,芒果蛋糕，很新鲜,核桃酥,铁板烧,香妃椰奶,很喜欢。儿子很喜欢吃,送了花，餐具好。,不错的俄，,雪山栗园,完全变形了,制作工艺精美，口感很好！,。。。蛋糕,真的不错！,榴莲芒返双拼,蛋糕奶茶,梳芒双拼,桂圆稣,榴莲、芒果双拼,半整体,榴恋草莓,同上,光盘,流口水了吧,水果布丁,六件套,超级芒果蛋糕,大盆花菜,榴芒慕絲,榴芒二重奏,怀旧炒粉,我的女儿,我大儿子,水果罐头,皇冠蛋糕,草莓果酱,羊肉串,圣诞节蛋糕,冰淇淋,巧克力新地,新榴芒双拼,巧克力黑森林,草莓黑森林,麻辣香锅,榴莲干,黑咖啡,白巧克力卷,甜心草莓派,核桃斯诺,午茶时光,特色蛋糕,纯巧克力,鲜榨果汁,十寸蛋糕,面包底,忌廉蛋糕,榴莲布丁,日本豆腐,菠萝蛋糕,咖啡慕斯蛋糕,抹茶拿破仑,酥皮包,芒果冰淇淋,纯牛奶,榴莲香草,巧克力味蛋糕,拿破仑酥,冰激淋蛋糕,爆米花,冰激凌蛋糕,西瓜汁,金桔柠檬,爱尔兰百利甜,小章鱼,花生米,自制冰激凌,萝卜白菜,巧克力口味蛋糕,手指饼,美食咖啡,老式蛋糕,榴莲披萨,百利甜情人,情深似海,鸡蛋糕,抹茶千层,水果草莓,牛肉粒,鲜奶蛋糕,芒果汁,手抓饼,大披萨,芒果爽,奶油冰淇淋,卡通生日蛋糕,巧克力华夫饼,海绵蛋糕,芒果巧克力提拉米苏,巧克力酱,芒果味冰淇淋蛋糕,蓝莓果酱,酸奶优格,马卡龙饼干,早生贵子,巧克力奶油,卡通系列,白葡萄酒,小面包,芒果生日蛋糕,星座蛋糕,绵绵冰,一小块蛋糕,蛋糕皮,榴莲往返,圣女果,哈根达斯冰淇淋,小点心,三角披萨,芝士榴莲,情迷黑森林,曲奇饼干,mini蛋糕,蓝莓酱,黑森林巧克力,奇异果,巧克力饼干,磅蛋糕卷,三明治,榴莲双拼慕斯蛋糕,树莓优格慕斯,魔鬼的诱惑,榴莲班戟,芒果酸奶雪糕,榴莲千层+芒果拿破仑,巴西咖啡,原味珍珠奶茶,疯狂烤翅,面包糕,绿豆沙,奶油水果蛋糕,芒榴双拼蛋糕,芝士榴莲蛋糕,榴莲蛋糕榴莲满满,芒果提拉米苏,水果夹心,土豆泥,白巧克力蛋糕,海盐脆脆,芭比公主,全心全意,银耳莲子汤,盛夏光年,奶油慕斯,鹏程万里,盛放鲜果,老鼠爱大米,天使蛋糕,小糕点,榴莲千层+芒果千层,牛奶片,卡布其诺,巧克力榛果,水果拼盘,步步高升,动物奶油蛋糕,水果夹心蛋糕,黑樱桃,黑森林樱桃,榴莲冰,芒果忌廉,欧式蛋糕,波萝包,浓情蜜意,5张女王蛋糕卡,猕猴桃,卡通蛋糕,西点蛋糕,巧克力蓝莓,裸蛋糕,芒果栗子蛋糕,大面包,芒果慕思,咖啡巧克力蛋糕,杯子蛋糕,花生酥,快乐时光,巧克力草莓,鲜草莓,抹茶慕斯,棉花糖,提拉米苏巧克力,巧克力饼,香草冰淇淋,戚风蛋糕,奶油酥,抹茶蜜豆,芒果雪糕,拿破仑饼,三层蛋糕胚,酸奶蛋糕,抹茶之森,紫薯布丁,芒果酸奶,巧克力慕斯,杏仁片,哈密瓜,芒果布丁蛋糕,单恋黑森林,烤肉比萨,千层蛋糕皮,提拉米酥,Tiramisu,绿茶奶盖",
  "group_info": 3,
  "has_group": true,
  "history_coupon_count": 887080,
  "introduction": "在深圳、湖南拥有30多间BLISS CAKE幸福西饼直营店，产品质量有目共睹。以新鲜制作为品牌核心竞争力，不断创新，品种繁多，各具风味，深深地融入每个顾客心中。用心去感受顾客的需求，为消费者生产出健康、安全、绿色的烘焙食品，让更多普通消费者享受高品质的烘焙食品。",
  "is_hot": false,
  "is_snack": false,
  "is_support_appointment": false,
  "is_waimai": true,
  "lat": 22.583933,
  "latest_week_coupou": 67,
  "lng": 114.133347,
  "lowest_price": 39.9,
  "mall_id": 0,
  "mark_numbers": 174856,
  "name": "幸福西饼生日蛋糕(布心店)",
  "open_info": "周一至周日
全天",
  "phone": "010-89171717",
  "poi_id": 618024,
  "preferent": false,
  "reference_price": 0,
  "show_type": "food",
  "style": "随便吃吃",
  "subway_station_id": "711,",
  "wifi": true,
}
`;
